### PR TITLE
Zbulu split

### LINKED ISSUE #5958: [Source Request] <mytoon>
## Source website
- Name: mytoon
- Link: https://mytoon.net/
- Language(s): english

### PR DESCRIPTION
**Pull request by jopejoe1**
Split `Zbulu`
removed `HeavenManga`
added `My Toon`
as `HeavenManga` redirects to `My Toon`

Closes #5958

### CODE REVIEW

File: src/all/zbulu/src/eu/kanade/tachiyomi/extension/all/zbulu/ZbuluFactory.kt [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-        KooManga(),
-        BuluManga(),
-    )
-}
-
-class HolyManga : Zbulu("HolyManga", "https://w15.holymanga.net")
-class HeavenManga : Zbulu("HeavenManga", "https://heaventoon.com")
-class KooManga : Zbulu("Koo Manga", "https://ww1.koomanga.com")
-class BuluManga : Zbulu("Bulu Manga", "https://ww5.bulumanga.net") {
-    override fun pageListParse(document: Document): List<Page> {
```
Comment (Riztard) [inline src/all/zbulu/src/eu/kanade/tachiyomi/extension/all/zbulu/ZbuluFactory.kt:21]: is the new `bulu manga` doesnt need this override?

Comment (jopejoe1) [reply src/all/zbulu/src/eu/kanade/tachiyomi/extension/all/zbulu/ZbuluFactory.kt:21]: I tried it in TachiyomiSY with out the override and the manga's i tested it one worked fine i can re add it if you want to.

Comment (Riztard) [reply src/all/zbulu/src/eu/kanade/tachiyomi/extension/all/zbulu/ZbuluFactory.kt:21]: just want to make sure

**Comment by CarlosEsco** [comment]
Thanks for the work.  Feel free to keep splitting the remaining extensions :smile:

**Pull request merged**